core/src/summary.cc

```diff
@@ -3,18 +3,21 @@
 namespace prometheus {
 
 Summary::Summary(const Quantiles& quantiles,
-                 const std::chrono::milliseconds max_age, const int age_buckets)
-    : quantiles_{quantiles},
-      count_{0},
-      sum_{0},
-      quantile_values_{quantiles_, max_age, age_buckets} {}
-
-void Summary::Observe(const double value) {
+                 const std::chrono::milliseconds& max_age,
+                 const int& age_buckets)
```

Comment:
Not sure if this change is necessary (to minimize modifications) --> in particular `const int& age_buckets` versus `const int age_buckets`

Reply:
You caught me. I can change it back but I'll wait to see what the other reviewers say.